CONTRIBUTING.md

```diff
@@ -47,15 +47,28 @@ Create a branch with a descriptive name, such as `add-search`.
 
 ### Step 5: Push your feature branch to your fork
 
-We use [semantic-versioning](https://semver.org/) and the [conventional commit message format](https://www.conventionalcommits.org/en/v1.0.0/). Keep a separate feature branch for each issue you want to address. As you develop code, continue to push code to your remote feature branch. Example:
+We use [semantic-versioning](https://semver.org/) and the [conventional commit message format](https://www.conventionalcommits.org/en/v1.0.0/). Keep a separate feature branch for each issue you want to address. As you develop code, continue to push code to your remote feature branch. 
+
+**Important:** When modifying or adding features for a specific part of the SDK, commits and pull requests should include a scope. The scope must be either `boxsdk` or `boxsdkgen`. For general changes that don't affect a specific part (e.g., documentation updates, build configuration), the scope is optional. Examples:
```

Comment:
> The scope must be either `boxsdk` or `boxsdkgen`

How can contributor determine which one to use? Shouldn't we only allow `boxsdk` and mention only part of the SDK can accept contributions (manual one) and the generated one cannot (PR still can be opened I guess)?